src/components/Navbar/ProfileDropdown.jsx

```diff
@@ -11,15 +11,23 @@ import {
   MenuItem,
   MenuDivider,
   Text,
+  Button,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Button' is defined but never used.